indy_node/test/pool_restart/test_pool_restart.py

```diff
@@ -40,6 +42,24 @@ def test_pool_restart(
     _comparison_reply(responses, req_obj)
 
 
+def test_restarter_can_initialize_after_pool_restart(txnPoolNodeSet):
+    '''
+    1. Schedule restart after restart_timeout seconds
```

Comment:
Seems like after test simplification this item is obsolete?

Reply:
Sorry, yes. Fixed it.